packages/snaps-sdk/src/types/handlers/user-input.ts

```diff
@@ -54,6 +55,19 @@ export const ButtonClickEventStruct = assign(
  */
 export type ButtonClickEvent = Infer<typeof ButtonClickEventStruct>;
 
+export const AccountSelectorStateStruct = object({
+  accountId: string(),
```

Comment:
Is this meant to always be available? Wouldn't that "leak" the account ID to Snaps that don't manage the account?

Reply:
We agreed that it's was okay to always have the account ID available, it's not sensitive data IIRC. We could hide it but that adds a case to handle when constructing state (which is already pretty complex)